main_test.go

```diff
@@ -48,3 +50,75 @@ func TestCafeWhenOk(t *testing.T) {
 		assert.Equal(t, http.StatusOK, response.Code)
 	}
 }
+
+// tasks
+const cityTest = "tula"
+
+// TestCafeCount() — проверяет работу сервера при разных значениях параметра count
+func TestCafeCount(t *testing.T) {
+	handler := http.HandlerFunc(mainHandle)
+
+	requests := []struct {
+		count int // передаваемое значение count
+		want  int // ожидаемое количество кафе в ответе
+	}{
+		{0, 0},
+		{1, 1},
+		{2, 2},
+		{100, len(cafeList[cityTest])},
```

Comment:
А вдруг в какой-то момент кафе станет больше 100?
```suggestion
		{100, min(len(cafeList[cityTest]), 100)},
```